include/SimCore/UserAction.h

```diff
@@ -188,9 +188,9 @@ class UserAction {
 
 }  // namespace simcore
 
-#define DECLARE_ACTION(NS, CLASS)                                        \
-  namespace {                                                            \
-    auto v = ::simcore::UserAction::Factory::get().declare<NS::CLASS>(); \
+#define DECLARE_ACTION(NS, CLASS)                                             \
```

Comment:
This change here just makes it so multiple user actions can be declared in one file. It just changes the name of the generated variable to depend on the class name

Reply:
makes sense, good idea